### PR TITLE
Primus display status service [stage-2]

### PR DESCRIPTION
**Pull request by settinghead**

### CODE REVIEW

**Comment by settinghead** [comment]
@ezequielc @alex-deaconu the display status obtained from Messaging Service now logs in the console when you navigate to display list page. Please take a look at the approach in the PR and see if you find any issues.

If everything looks good, I'll be implementing the UI part in the next card.

Thanks!

**Comment by settinghead** [comment]
@alex-deaconu @ezequielc oops please hold off the review. There seems to be a uglify issue.

**Comment by ezequielc** [comment]
@settinghead You may have missed my comment in the card. Instead of pointing to the js file from `https://display-messaging.risevision.com:3000/primus/primus.js`, @tejohnso suggested that we copy it to Apps and use it from here. Thoughts? Thanks!

**Comment by settinghead** [comment]
@ezequielc indeed missed. Will do.

**Comment by alex-deaconu** [comment]
@settinghead code looks good. However, tests need to be added (that's why the build fails). Thanks.

**Comment by settinghead** [comment]
@ezequielc @alex-deaconu code updated and unit test added. You can test it at: https://apps-stage-0.risevision.com

File: test/unit/displays/services/svc-display.tests.js [hunk 1]
```diff
@@ -442,4 +472,20 @@ describe('service: display:', function() {
         .then(null,done);
     });
   });
+
+  var getDisplayStatus;
+  describe('primus', function() {
```
Comment (alex-deaconu) [inline test/unit/displays/services/svc-display.tests.js]: @settinghead I would prefer if `Primus` tests were added as a separate file (`svc-primus.tests.js`). Thanks.

**Comment by settinghead** [comment]
@alex-deaconu updated. Primus tests moved to a separate file.

**Comment by alex-deaconu** [comment]
@settinghead looks good. My suggestion is however to hold off merging this PR until the rest of the functionality (UI part) has been implemented. Also, please don't forget to squash the commits. Thanks.

**Comment by settinghead** [comment]
closing as this PR is now part of https://github.com/Rise-Vision/rise-vision-apps/pull/270

**Pull request closed without merging**